File: Lista 5/L5_Q9.c
#include <stdio.h>

int main(void){
    
    float valor = 100, valor_desc;

    printf("Informe o valor da venda: ");
    scanf("%f", &valor);
    valor_desc = valor - valor*0.1;

    printf("\nTotal da pagar com discondo de 10%% eh R$%.2f\n", valor_desc);
    printf("O valor de cada parcela, 3x sem juros eh R$%.2f\n", valor / 3);
    printf("A comissao do vendedor, na venda a vista eh R$%.2f\n", valor_desc * 0.05);
    printf("A comissao do vendedor, na venda parcelada eh R$%.2f\n", valor * 0.05);

    printf("\n\n");

}

File: Período_1/Lista 5/Python/L5_Q9.py
valor = float(input("Informe o valor do produto: "))

print(f"O total a pagar com desconto de 10% é R${valor * 0.9: .2f}")

print(f"O valor de cada parcelo, no parcelamento de 3x sem jutos é R${valor / 3: .2f}")

print(f"A comissão do venderdor, se a compra for a vista é R${(valor * 0.9)*0.05: .2f}")

print(f"A comissão do venderdor, se a compra for parcelada é R${valor*0.05: .2f}")